internal/telemetry/zerolog_exporter.go

```diff
@@ -26,11 +28,22 @@ func (e *loggerExporter) ExportSpans(ctx context.Context, spans []trace.ReadOnly
 		parentId := span.Parent().SpanID()
 		statusCode := span.Status().Code
 		statusMsg := span.Status().Description
+
 		t := e.logger.Trace().
 			Str("trace_id", traceId).
 			Str("span_id", spanId).
+			Str("request_id", logging.EdgeRequestId(ctx)).
+			Str("org_id", identity.Identity(ctx).Identity.OrgID).
+			Str("account_number", identity.Identity(ctx).Identity.AccountNumber).
+			Str("job_id", logging.JobId(ctx)).
+			Str("job_type", logging.JobType(ctx)).
 			Dur("duration", duration)
 
+		accountId := identity.AccountIdOrZero(ctx)
+		if accountId != 0 {
+			t = t.Int64("account_id", accountId)
+		}
+
```

Comment:
This is the gist of the patch - I need to have in our zerolog exporter (which is used on stage/prod) request id which I can use to correlate all the records. It was not present at all, only trace_id which is unfortunately different because we cannot use it between api and worker pods (it gets closed).

Reply:
The job_id and job_type should be optional (set only when available) IMO, not to set them from all processes, having them blank may confuse us.

Reply:
Yeah, rebased. All fields are now compared and only added when needed.